external/source/shellcode/windows/x64/src/block/block_api.asm

```diff
@@ -19,95 +19,95 @@
 ; Note: This function is unable to call forwarded exports.
 
 api_call:
-  push r9                  ; Save the 4th parameter
-  push r8                  ; Save the 3rd parameter
-  push rdx                 ; Save the 2nd parameter
-  push rcx                 ; Save the 1st parameter
-  push rsi                 ; Save RSI
-  xor rdx, rdx             ; Zero rdx
-  mov rdx, [gs:rdx+96]     ; Get a pointer to the PEB
-  mov rdx, [rdx+24]        ; Get PEB->Ldr
-  mov rdx, [rdx+32]        ; Get the first module from the InMemoryOrder module list
-next_mod:                  ;
-  mov rsi, [rdx+80]        ; Get pointer to modules name (unicode string)
-  movzx rcx, word [rdx+74] ; Set rcx to the length we want to check 
-  xor r9, r9               ; Clear r9 which will store the hash of the module name
-loop_modname:              ;
-  xor rax, rax             ; Clear rax
-  lodsb                    ; Read in the next byte of the name
-  cmp al, 'a'              ; Some versions of Windows use lower case module names
-  jl not_lowercase         ;
-  sub al, 0x20             ; If so normalise to uppercase
-not_lowercase:             ;
-  ror r9d, 13              ; Rotate right our hash value
-  add r9d, eax             ; Add the next byte of the name
-  loop loop_modname        ; Loop untill we have read enough
+  push r9                     ; Save the 0x4th parameter
```

Comment:
did you mean to make the parameter comments hex? (no biggie, just thought it was funny)

Reply:
No, I applied a regex to the source code matched these while converting the decimal representation to hex for compatibility with the keystone engine. I thought I pushed up the fix for this but I think I lost it while rebasing.

Reply:
I prefer this.

Reply:
/s :)